### PR TITLE
getting stared ios

### PR DESCRIPTION
**Pull request by corinnekrych**
@cvasilak @lfryc here is a getting started with iOS, mind to review?

### CODE REVIEW

File: docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+== Get Started with iOS platform
+
+If you're brand new to iOS, this first section will give you interesting links to start with iOS environment and platform. We've all been a beginner at one point, do not fear iOS platform is easy to start with specially with its new Swift language.
+
+== Pre-requisites
+- Get a mac and an iOS device. 
+- Target the latest release: install Yosemite, Xcode 6.1.1. iOS 8.1.3
+
+== Online Training
+If you like screencast and MOOC session, I'd recommand the excellent Stanford classes. You can find them for free on link:https://itunes.apple.com/us/course/developing-ios-8-apps-swift/id961180099[iTunes].
```
Comment (cvasilak) [inline docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc]: s/recommand/recommend

File: docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+== Books
+There are plenty of good books to learn from, among goog one here is a book with tutorials from link:http://www.raywenderlich.com/store/ios-apprentice[Ray Wenderlich].
```
Comment (cvasilak) [inline docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc]: s/goog/good

File: docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+=== AeroGear: Xcode and iOS
+
+==== AeroGear 2.X
+Our libraries for AeroGear iOS 2.X are Swift based you will need at least Xcode 6.1.1.
+
+Runtime for Swift based demo and libraries is 8.X.
+
+==== AeroGear 1.6.X
+Our earlier version work from Xcode 5 and iOS 6.
```
Comment (cvasilak) [inline docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc]: s/from/for

File: docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+==== AeroGear 2.X
+Our libraries for AeroGear iOS 2.X are Swift based you will need at least Xcode 6.1.1.
+
+Runtime for Swift based demo and libraries is 8.X.
+
+==== AeroGear 1.6.X
+Our earlier version work from Xcode 5 and iOS 6. 
+
+**Note** Althgouh still supported AeroGear 1.6.X is a deprecated branch, all new features goes in 2.X version (on master for all libraries).
```
Comment (cvasilak) [inline docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc]: s/Althgouh/Although

Comment (cvasilak) [inline docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc]: s/goes/go

File: docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+=== Build with cocoapods
+
+Our Swift and ObjC libraries are build using Cocoapods 0.36.0.
+In your Podfile add the AeroGear dependecy you're intersted in (here for example OAuth2):
```
Comment (cvasilak) [inline docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc]: s/intersted/

File: docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+```bash
+pod 'AeroGearOAuth2'
```
Comment (cvasilak) [inline docs/guides/aerogear-ios-2.X/GettingStartedWithiOS.asciidoc]: I think we should we add a note for ```use_frameworks!``` required flag

**Comment by lfryc** [comment]
:heart: love this guide

perhaps we could add a link guiding people to `/ios/` landing page at the bottom?

**Comment by corinnekrych** [comment]
@cvasilak sorry for type (I should have used a syntax corrector :/) i fixed those + adding a note on use_frameworks!
@lfryc I've added the link to ios main page

**Comment by lfryc** [comment]
:+1: Awesome, thanks @corinnekrych , merging

**Comment by lfryc** [comment]
:rocket: landed: https://github.com/aerogear/aerogear.org/commit/d4f6f1b449241a431b88ce8db31696bf4509f7a4

**Pull request closed without merging**